classes/interaction.lua

```diff
@@ -95,7 +102,18 @@ function Interaction:drawSprite()
         local scale = 0.025 * (distanceRatio)
         local dict = defaultIndicator.dict
         local txt = defaultIndicator.txt
-        DrawInteractiveSprite(dict, txt, 0, 0, scale, scale * ratio, 0.0, color.x, color.y, color.z, color.w)
+        local spriteColour = color
+
+        if self.sprite ~= nil then
+            dict = self.sprite.dict
```

Comment:
we will run into issues here if they define a color and not a sprite

Reply:
Updated in commit [f1c7f93](https://github.com/Sleepless-Development/sleepless_interact/pull/7/commits/f1c7f9380bf5f94cd95dcd7aba6c1c55d00c2f44)